app/serializers/articles_index_serializer.rb

```diff
@@ -1,5 +1,6 @@
 class ArticlesIndexSerializer < ActiveModel::Serializer
-  attributes :id, :title, :teaser, :date, :category, :image, :rating, :author, :premium, :published, :comments
+  attributes :id, :title, :teaser, :date, :category, :language, :image, :rating, :author, :premium, :published,
+             :comments
```

Comment:
lol what happened here? 😅

Reply:
lel looks strange only in the file changes for me 🤷‍♂️

Reply:
magic